### PR TITLE
Additional fix for #1269. Also mapping /dist/dev/css/ to /css/ in index.html

### PR DESCRIPTION
**Pull request by sasikumardr**
fix for rendering css file in original commit related to #1269, moving rewrite rules to inline of PLUGIN_CONFIGS, mapping /dist/dev/css/ to /css/ in index.html for consistency.

### CODE REVIEW

File: tools/config/seed.config.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { join } from 'path';
 import { argv } from 'yargs';
+import * as slash from 'slash';
```
Comment (TheDonDope) [inline tools/config/seed.config.ts]: Can you please order the imports alphabetically to be consistent with the [Styleguide](https://angular.io/styleguide#!#03-06)

**Comment by mgechev** [comment]
Once you address the comment of @TheDonDope, we'll be ready to merge.

**Comment by mgechev** [comment]
@TheDonDope would you approve?

**Comment by mgechev** [comment]
@sasikumardr last thing - would you rebase against master?

**Comment by sasikumardr** [comment]
I'm going on a vacation trip for 4 days and will not have computer access. I'll update pull request as soon as I reach back. Or if someone else interested can take care of the change.

**Pull request merged**